src/django_upgrade/fixers/admin_register.py

```diff
@@ -0,0 +1,115 @@
+"""
+Replace `admin.site.register` with the new `@register` decorator syntax:
+https://docs.djangoproject.com/en/dev/releases/1.7/#minor-features
+"""
+from __future__ import annotations
+
+import ast
+from functools import partial
+from typing import Iterable, MutableMapping
+from weakref import WeakKeyDictionary
+
+from tokenize_rt import Offset, Token
+
+from django_upgrade.ast import ast_start_offset
+from django_upgrade.data import Fixer, State, TokenFunc
+from django_upgrade.tokens import erase_node, extract_indent, insert
+
+fixer = Fixer(
+    __name__,
+    min_version=(1, 7),
```

Comment:
added 1.7 and 1.8 to `--target-version` options so that it makes sense to have a 1.7+ fixer